sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -292,6 +295,9 @@ func newLanguageHost(
 }
 
 func (host *nodeLanguageHost) connectToEngine() (pulumirpc.EngineClient, io.Closer, error) {
+	if host.engineAddress == "" {
+		fmt.Fprintln(os.Stderr, "Warning: no engine address is set, logging will not work")
```

Comment:
Same comment as python, I think we can just make this an error.

Reply:
done